tile_id_pass/tile_id_pass.py

```diff
@@ -0,0 +1,58 @@
+from gemstone.common.transform import replace, Generator, FromMagma
+from io_core.io_core_magma import IOCore
+from canal.interconnect import Interconnect
+from gemstone.common.configurable import Configurable, ConfigurationType
+import magma
+import mantle
+from gemstone.generator.const import Const
+from hwtypes import BitVector
+
+## We create constant tie hi and tie lo outputs on each tile
```

Comment:
You only need one `#`, otherwise it will fail `pycodestyle`. Not sure why the stylecheck is not running.